python/ray/util/debugpy.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import os
+import sys
+import threading
+import importlib
+
+import ray
+from ray.util.annotations import DeveloperAPI
+
+log = logging.getLogger(__name__)
+
+POST_MORTEM_ERROR_UUID = "post_mortem_error_uuid"
+
+
+def _try_import(module):
+    try:
+        return importlib.import_module(module)
+    except ModuleNotFoundError:
+        log.error(
+            f"Module '{module}' cannot be loaded. "
+            f"Ray Debugger will not work without '{module}'. "
+            f"Install this module using 'pip install {module}' "
```

Comment:
make this specific, look for debugpy >= 1.8.0, and then raise warning if not found (and suggest to user to install debugpy==1.8.0)